hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -172,9 +172,9 @@
       new Bytes(Bytes.toBytes(NORMALIZER_TARGET_REGION_COUNT));
 
   @InterfaceAudience.Private
-  public static final String NORMALIZER_TARGET_REGION_SIZE = "NORMALIZER_TARGET_REGION_SIZE";
+  public static final String NORMALIZER_TARGET_REGION_SIZE_MB = "NORMALIZER_TARGET_REGION_SIZE_MB";
```

Comment:
Suggest you retain support of the old name too.

Reply:
@saintstack Thanks for pointing out. Done

Reply:
I think you have not retained support for the old key. I expected to see both `"NORMALIZER_TARGET_REGION_SIZE"` and `"NORMALIZER_TARGET_REGION_SIZE_MB"` used in the method `getNormalizerTargetRegionSize()`.

Reply:
Yeah, seems to missed this fact. Fixed now. Thanks